csrc/id_model/indexing_utils.h

```diff
@@ -55,5 +55,27 @@ inline IterDomain* getLoopPromotion(
   return loop_promotion_map_it->second;
 }
 
+// Check if unswitching a given for-loop actually matters. For example,
```

Comment:
@zasdfgbnm Forgot to add this code. I already merged this but let me know if you have any concern.